pokemongo_bot/cell_workers/complete_tutorial.py

```diff
@@ -179,3 +179,36 @@ def _set_tutorial_state(self, completed):
         except KeyError:
             self.logger.error("KeyError while setting tutorial state")
             return False
+
+    def _set_team(self):
+        if self.team == 0:
```

Comment:
At `initialize`:
`self.team = self.config.get('team',0)`
If it is 0 then no team must be picked.